### PR TITLE
Cat sprite revert, attempt 3

### PR DESCRIPTION
**Pull request by silencer-pl**
# About the pull request

This PR is #2258 but without conflicts in the dmi files that github refuses to resolve otherwise. The rest is copied and pasted from there:

This is essentially reopened #2020  , but now with extras.

For a full breakdown why the sprites deserve a revert, please consult my treatise found in #2020 TLDR: The new sprites are cartoony and anatomically incorrect and clash with CM's fairly realistic art style. Also they turned Jones into a tabby, unacceptable.

In addition to reverting the sprites, this does the following:

- Adds an inhand sprite for Jones/calicos, we were missing that for some reason. 
- Organizes the cat holders so they make sense.
- For consistency's sake adds a generic black cat mob type along Runtime. 
- Removes being able to gib cats using ATM machines, apparently added as a funny meme reference. Killing Jones indiscriminately is already an issue to the point that it was sanctioned OOCly, having a meme way of doing that is not helping anything. Also, it is incredibly upsetting to know we even had that in the code in the first place.

# Explain why it's good for the game

Sprite consistency good.
Code consistency good.
Meme mechanics that are essentially ooc rule breaks bad.

Also _all_ cats are best cats. 

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:Silencer_pl
imageadd: Older, more anatomically correct and less cartoony cat sprites have been readded. Jones is a calico again!
del: Cats can no longer be gibbed in ATMs.
/:cl:

### CODE REVIEW

File: code/modules/mob/holder.dm [hunk 1]
```diff
@@ -64,25 +64,27 @@
...
 	desc = "A domesticated, feline pet. Has a tendency to adopt crewmembers."
 	icon_state = "cat2"
 
-/obj/item/holder/blackcat
+/obj/item/holder/cat/kitten
+	name = "kitten"
+	desc = "D'aaawwww"
+
+/obj/item/holder/cat/Jones
+	name = "Jones"
```
Comment (Zonespace27) [inline code/modules/mob/holder.dm:77]:
```suggestion
	name = "\improper Jones"
```

File: code/modules/mob/holder.dm [hunk 2]
```diff
@@ -64,25 +64,27 @@
...
 	name = "black cat"
 	desc = "It's a cat, now in black!"
 	icon_state = "cat"
 
-/obj/item/holder/Jones
-	name = "Jones"
-	desc = "A tough, old stray whose origin no one seems to know."
-	icon_state = "cat2"
+/obj/item/holder/cat/blackcat/Runtime
+	name = "Runtime"
```
Comment (Zonespace27) [inline code/modules/mob/holder.dm:86]:
```suggestion
	name = "\improper Runtime"
```

**Comment by silencer-pl** [comment]
Yes I clicked them separately instead of creating a batch. Yes, I am an idiot, why do you ask? :P

**Review by nauticall — APPROVED** [review]
sprites good

**Pull request merged**